app/models/task.py

```diff
@@ -3,3 +3,23 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None)
```

Comment:
Per the SqlAlchemy [docs](https://docs.sqlalchemy.org/en/13/core/metadata.html#sqlalchemy.schema.Column.params.nullable), when nullable is True (which is the default value so we don't need explicitly add the constraint) then the value will be NULL in the db. This means that you do not need default=None because that is already the default behavior. 

If line 8 was written like `completed_at = db.Column(db.DateTime, default=None)` then that would mean it is nullable and it would have a. null value in the DB and when you query for a Task object then the value of the completed_at attribute would be None.